python/ray/dag/input_node.py

```diff
@@ -17,45 +17,52 @@ class InputNode(DAGNode):
     entrypoints, but only one instance of InputNode exists per DAG, shared
     among all DAGNodes.
 
-    >>> Example:
-    >>>            m1.forward
-    >>>            /       \
-    >>>    dag_input     ensemble -> dag_output
-    >>>            \       /
-    >>>            m2.forward
+    Example:
+
+    .. code-block::
+
+                m1.forward
+                /       \
+        dag_input     ensemble -> dag_output
+                \       /
+                m2.forward
 
     In this pipeline, each user input is broadcasted to both m1.forward and
     m2.forward as first stop of the DAG, and authored like
 
-    >>> @ray.remote
-    >>> class Model:
-    ...     def __init__(self, val):
-    ...         self.val = val
-    ...     def forward(self, input):
-    ...         return self.val * input
-
-    >>> @ray.remote
-    >>> def combine(a, b):
-    ...     return a + b
-
-    >>> with InputNode() as dag_input:
-    >>>     m1 = Model.bind(1)
-    >>>     m2 = Model.bind(2)
-    >>>     m1_output = m1.forward.bind(dag_input[0])
-    >>>     m2_output = m2.forward.bind(dag_input.x)
-    >>>     ray_dag = combine.bind(m1_output, m2_output)
-
-    >>> # Pass mix of args and kwargs as input.
-    >>> ray_dag.execute(1, x=2) # 1 sent to m1, 2 sent to m2
-
-    >>> # Alternatively user can also pass single data object, list or dict
-    >>> # and access them via list index, object attribute or dict key str.
-    >>> ray_dag.execute(UserDataObject(m1=1, m2=2))
-    ...     # dag_input.m1, dag_input.m2
-    >>> ray_dag.execute([1, 2]))
-    ...     # dag_input[0], dag_input[1]
-    >>> ray_dag.execute({"m1": 1, "m2": 2})
-    ...     # dag_input["m1"], dag_input["m2"]
+    .. code-block:: python
```

Comment:
Using `code-block:: python` so we can track this as an untested/broken example that needs to be updated.